### PR TITLE
Remove abstract class

### PR DESCRIPTION
**Pull request by becca-bailey**

### CODE REVIEW

File: src/main/java/application/game/Computer.java [hunk 1]
```diff
@@ -5,28 +5,25 @@
...
-            board[computerMove] = marker;
-        }
+        int computerMove = getMove();
+        board[computerMove] = marker;
+
         return board;
     }
 
-    public int getComputerMove() {
+    public int getMove() {
```
Comment (rmulhol) [inline src/main/java/application/game/Computer.java]: Should this be a public method?

Comment (tommcgee) [reply src/main/java/application/game/Computer.java]: @rmulhol It definitely doesn't have to be, right now we're testing `getBoard()` and `getMove()` separately.

Comment (rmulhol) [reply src/main/java/application/game/Computer.java]: Cool. I might consider testing `getMove()` through `getBoard()` if that's the only caller, and then adding that private declaration to `getMove()`. That way, you've got a narrower public interface, and you're not tempting future developers to reach into the internals of your object.

Comment (tommcgee) [reply src/main/java/application/game/Computer.java]: @rmulhol We went ahead and followed your suggestion, `getMove()` is now private. Very good point about future developers.

**Pull request merged**